test/packages/parallel/oracle/data_stream/memory/_dev/test/system/test-memory-config.yml

```diff
@@ -0,0 +1,16 @@
+vars:
+  hosts:
+    - "oracle://sys:Oradoc_db1@elastic-package-service-oracle-1:1521/ORCLCDB.localdomain?sysdba=1"
+agent:
```

Comment:
According to the tests in this PR, to update the `oracle` package in the integrations repository it should be needed:
- add this key `agent` to each of the test system configuration files
- remove `_dev/deploy/agent` folders
- replace agent folders with `_dev/deploy/docker` folder (something similar to this PR) with just the service definition (`oracle`).